src/Tokenizer/Manipulators/TokenRemover.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Manipulators;
```

Comment:
In `PhpCsFixer\Tokenizer` we have `Analyzer`, `Generator`, `Resolver` and `Transformer`, so let's make this `Manipulator`.